crates/sui-benchmark/src/workloads/adversarial.rs

```diff
@@ -354,7 +354,7 @@ impl WorkloadBuilder<dyn Payload> for AdversarialWorkloadBuilder {
         // Gas coin for publishing adversarial package
         let (address, keypair) = get_key_pair();
         vec![GasCoinConfig {
-            amount: MAX_GAS_FOR_TESTING,
+            amount: MAX_GAS_FOR_TESTING * 1000,
```

Comment:
Comment on the 1000 factor?

Reply:
Removed, will need to factor in computation cost and storage cost in a future PR. Updated PR description to reflect broken adversarial workloads